jsonschema-core.xml

```diff
@@ -123,44 +123,130 @@
+                    location, annotation keywords need to specify any unusual handling of
+                    multiple applicable occurrences of the keyword with different values.
+                    The default behavior is simply to collect all values.
+                </t>
+                <t>
+                    Unlike assertion results, annotation data can take a wide variety of forms,
+                    which are provided to applications to use as they see fit.  JSON Schema
+                    implementations are not expected to make use of the collected information
+                    on behalf of applications.
+                </t>
+                <t>
+                    While "short-circuit" evaluation is possible for assertions, collecting
+                    annotations requires examining all schemas that apply to an instance
+                    location, even if they cannot change the overall assertion result.
                 </t>
             </section>
 
-            <section title="Hypermedia and Linking">
+            <section title="Schema Vocabularies" anchor="vocabulary">
+                <t>
+                    A JSON Schema vocabulary is a set of keywords defined for a particular
+                    purpose.  The vocabulary specifies the meaning of its keywords as
+                    assertions, annotations, and/or any vocabulary-defined keyword category.
+                </t>
                 <t>
-                    JSON Hyper-Schema describes the hypertext structure of a JSON document.
-                    This includes link relations from the instance to other resources,
-                    interpretation of instances as multimedia data, and submission data required to
-                    use an API.
+                    Several vocabularies are provided as
+                    standards in this and closely related documents.  These vocabularies
+                    are used with the core keywords defined as fundamental to the
+                    "application/schema+json" media type.
                 </t>
                 <t>
-                    Hyper-schema behaviour and keywords are specified in
-                    <xref target="json-hyper-schema">a separate document</xref>.
+                    Schema authors are encouraged to define their own vocabularies for
+                    domain-specific concepts.  A vocabulary need not be a standard to
+                    be re-usable, although users of extension vocabularies MUST NOT
+                    assume that any JSON Schema implementation can support the vocabulary
+                    unless it specifically documents such support.
                 </t>
+                <section title="Subschema Application">
+                    <t>
+                        This vocabulary provides keywords for applying subschemas to the
+                        instance in various ways.  It is defined in this document, and
```

Comment:
It might be helpful to insert an `xref` pointing to "A Vocabulary for Applying Subschemas" section in this document here.